src/contractTest/java/uk/gov/hmcts/reform/managecase/consumer/McaExampleConsumerTest.java

```diff
@@ -0,0 +1,135 @@
+package uk.gov.hmcts.reform.managecase.consumer;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslJsonRootValue;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import com.google.common.collect.ImmutableMap;
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import org.apache.http.entity.ContentType;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(PactConsumerTestExt.class)
+public class McaExampleConsumerTest {
```

Comment:
again, repeating suggestion to split these classes into distinct ones for tests of individual operations.

Reply:
One more thing - are we required to write example consumer tests as the provider?

Reply:
not required as a standard. This is an example for XUI to start implementing their contracts.